ansible/roles/monasca/tasks/config.yml

```diff
@@ -27,6 +27,11 @@
   notify:
     - Restart monasca-api container
     - Restart monasca-log-api container
+    - Restart monasca-log-transformer container
+    - Restart monasca-log-persister container
+    - Restart monasca-thresh container
+    - Restart monasca-notification container
+    - Restart monasca-persister container
```

Comment:
Some other places in kolla use `Restart {{ item.key }} container`

Reply:
Fixed